app/views/curation_concerns/base/_find_work_widget.html.erb

```diff
@@ -0,0 +1,9 @@
+<input class="new-form-control string multi_value optional related_works_ids work_child_members_ids form-control multi-text-field ui-autocomplete-input"
+       data-autocomplete-url="/authorities/search/local/works"
```

Comment:
You should use the rails path helpers, so that it works if the application is not running at `/`